refactor: User 관련 코드 적용

jiyoon0000 opened the pull request.

jiyoon0000 committed:
refactor: User 관련 코드 적용

jiyoon0000 committed:
feat: 유저 워크스페이스 전체 조회

jiyoon0000 committed:
feat: 워크스페이스 단일 조회

jiyoon0000 committed:
feat: 워크스페이스 수정

alsqja commented on src/main/java/com/example/panttegi/workspace/entity/Workspace.java:
메서드 이름은 동사/전치사로 시작하는게 좋아서
updateWorkspace 나 updateUpdateFields 정도가 좋아보입니다

alsqja commented on src/main/java/com/example/panttegi/workspace/service/WorkspaceService.java:
데이터베이스를 조회할 때는 Transactional 처리가 필요 없을 거 같습니다.
Role 확인하는 부분 필터레벨에서 처리 예정입니다. 바로 빼실 필요는 없고 다음꺼 진행 하실 때 안하셔도 괜찮습니다.
getWorkspace 메서드에서 api 에 있는 boards 를 보내주는 부분이 있는지 모르겠습니다.

jiyoon0000 committed:
fix: 워크스페이스 수정
-워크스페이스 단일 조회시 보드 내용 나오게 수정
-메서드 이름 변경 -> updateWorkspace
-워크스페이스 조회 시 Transactional 삭제
